example_9/description/ros2_control/rrbot.ros2_control.xacro

```diff
@@ -0,0 +1,42 @@
+<?xml version="1.0"?>
+<robot xmlns:xacro="http://www.ros.org/wiki/xacro">
+
+  <xacro:macro name="rrbot_ros2_control" params="name prefix use_gazebo_classic:=^|false use_gazebo:=^|false">
+
+    <ros2_control name="${name}" type="system">
+      <hardware>
+        <xacro:if value="${use_gazebo_classic}">
+          <plugin>gazebo_ros2_control/GazeboSystem</plugin>
```

Comment:
maybe for fortress
`<plugin>ign_ros2_control/IgnitionSystem</plugin>` 
and onwards
<plugin>gz_ros2_control/IgnitionSystem</plugin>